app/enterprise/1.5.x/kong-for-kubernetes/index.md

```diff
@@ -1,14 +1,17 @@
 ---
 title: Kong for Kubernetes Enterprise
+toc: false
 ---
 
-Kong for Kubernetes Enterprise (K4K8S) is a Kubernetes Ingress Controller. A Kubernetes Ingress Controller is a proxy that exposes Kubernetes services from applications (e.g., Deployments, RepliaSets) running on a Kubernetes cluster to client applications running outside of the cluster. The intent of an Ingress Controller is to provide a single point of control for all incoming traffic into the Kubernetes cluster.
+Kong for Kubernetes Enterprise is a deployment of {{site.ee_gateway_name}} onto Kubernetes with an ingress controller, which is responsible for configuring Kong. A Kubernetes ingress controller is a proxy that exposes Kubernetes services from applications (e.g., Deployments, ReplicaSets) running on a Kubernetes cluster to client applications running outside of the cluster. The intent of an ingress controller is to provide a single point of control for all incoming traffic into the Kubernetes cluster.
 
-For example, in a common use case, if an application deployed to Kubernetes exposes an API that needs to be used by Web or mobile-client applications or services in another cluster, then a Kubernetes Ingress Controller is required. It is important for the Ingress Controller to secure and manage traffic according to various policies that can be changed on the fly based on the use-case and application.
+For example, here's a common use case: an application deployed to Kubernetes exposes an API that needs to be used by Web or mobile-client applications or services in another cluster. It uses a Kubernetes ingress controller, which can secure and manage traffic according to various policies that can be changed on the fly based on the use case and application.
 
-Kong for Kubernetes Enterprise stores all of the configuration in the Kubernetes datastore (etcd) using Custom Resource Definitions (CRDs), meaning you can use Kubernetes' native tools to manage Kong and benefit from Kubernetes' declarative configuration, RBAC, namespacing, and scalability. Also, because the configuration is stored in Kubernetes, no database needs to be deployed for Kong. Kong runs in in-memory mode, making it operationally easy to run, upgrade, and backup Kong.
-
-Kong for Kubernetes Enterprise natively integrates with the Cloud Native Computing Foundation (CNCF) eco-system to provide out of the box monitoring, logging, certificate management, tracing, and scaling.
+Here are some benefits of using Kong for Kubernetes Enterprise:
+* It stores all of the configuration in the Kubernetes datastore (etcd) using Custom Resource Definitions (CRDs), meaning you can use Kubernetes' native tools to manage Kong and benefit from Kubernetes' declarative configuration, RBAC, namespacing, and scalability.
+* Because the configuration is stored in Kubernetes, no database needs to be deployed for Kong. Kong runs in in-memory mode, making it operationally easy to run, upgrade, and back up.
```

Comment:
```suggestion
* Because the configuration is stored in Kubernetes, no database needs to be deployed for Kong. Kong runs in DB-less mode, making it operationally easy to run, upgrade, and back up.
```